run_local_with_dependencies.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-sm --start MONGO DATASTREAM EMAIL HMRC_EMAIL_RENDERER MAILGUN_STUB DESKPRO_STUB HMRCDESKPRO
+sm2 --start MONGO DATASTREAM EMAIL HMRC_EMAIL_RENDERER MAILGUN_STUB DESKPRO_STUB HMRCDESKPRO
```

Comment:
Not everyone is on sm2 so do not change this.  Alias it locally if you want to.

Reply:
Thought i'd excluded this change.